### ISSUE TITLE
No response with Openemu on Mac

### ISSUE THREAD

**Issue by Greeneied** [PJSoftCo/VGamepad#29]
First, thanks for building a Mac version. I’ve successfully installed the software on my mac and iphone. However when I try to use the controls on Openemu the game doesn’t respond. The game still responds to the keyboard, but not the iphone controls. I was able to type in the Notes app on my computer so that seems fine. 

I’m running mac os 12.7.4 and using iphone 14 pro max with the newest ios.

**Comment by pjsoftcoUser** [comment]
Have you mapped the iPhone controls to your keyboard keys just like the youtube instruction video? Right now it can only be mapped to the keyboard and mouse keys and not the gamepad since the xbox or PS controller driver emulation does not exist on Mac!

**Comment by Greeneied** [comment]
Yeah. I mapped the controls to the keyboard keys like a, b, the direction arrows, etc. I’m trying to play an NES game on openemu at the moment.

**Comment by Greeneied** [comment]
I also checked in Nestopia and I got it to do a couple things but not fully function.

**Comment by pjsoftcoUser** [comment]
Could you please explain a little more in details or provide a video?
